lib/npg_pipeline/function/stage2pp.pm

```diff
@@ -24,8 +23,6 @@ with 'npg_common::roles::software_location' =>
 
 Readonly::Scalar my $DEFAULT_PIPELINE_TYPE => q[stage2pp];
 
```

Comment:
The code refactoring for this function is not finished, hardcoded resource definitions should be moved to the resources definition for the respective functions